bootstrap/051_materialization_status.sql

```diff
@@ -0,0 +1,46 @@
+
+create or replace view internal.all_task_history(run, success, input, output, table_name) as
+    select run, success, input, output, 'QUERY_HISTORY' from internal.task_query_history
+    union all
+    select run, success, input, output, 'WAREHOUSE_EVENTS_HISTORY'  from internal.task_warehouse_events;
```

Comment:
I didn't want to explicitly create this but I was struggling with a way to compose this from the `task_tables` CTE below.

I feel like I should be able to use a static list of task history tables (internal.task_query_history, internal.task_warehouse_events) and compose an equivalent `select` statement without dynamically creating sql.

Any thoughts?

Reply:
Solution came to me once I stopped looking at the computer -- just push this query into the CTE and join on 'table_name'. 🤦🏼‍♀️